CMakeLists.txt

```diff
@@ -36,22 +34,26 @@ cet_set_compiler_flags(DIAGS CAUTIOUS
   EXTRA_FLAGS -pedantic -I $ENV{CLANG_FQ_DIR}/include/c++/v1
 )
 
-cet_report_compiler_flags()
+cet_report_compiler_flags(REPORT_THRESHOLD VERBOSE)
 
 # these are minimum required versions, not the actual product versions
-find_ups_product( larcoreobj )
-find_ups_product( larcorealg )
-find_ups_product( lardataobj )
-find_ups_product( lardataalg )
-find_ups_product( nusimdata  )
-find_ups_product( canvas     )
-find_ups_product( cetlib     )
-find_ups_product( clhep      )
-find_ups_product( guideline_sl )
-find_ups_root()
-find_ups_boost()
-
-# macros for dictionary and simple_plugin
+find_package( icarusalg )
+find_package( larcorealg )
+find_package( larcoreobj )
+find_package( lardataalg )
+find_package( lardataobj )
+find_package( nusimdata  )
+find_package( dk2nudata )
+find_package( cetlib )
+find_package( cetlib_except )
+find_package( canvas     )
+find_package( canvas_root_io )
+find_package( messagefacility )
+find_package( ROOT )
+find_package( Boost COMPONENTS unit_test_framework)
+find_package( CLHEP )
+find_package( Microsoft.GSL HINTS $ENV{GUIDELINE_SL_DIR} )
```

Comment:
These will need `EXPORT` keywords added if they satisfy link _or_ header-level dependencies on exported (_i.e._ not test) libraries or headers.